shuffle/app/(frontend)/polls/[slug]/page.tsx

```diff
@@ -38,18 +38,35 @@ async function getData({ params }: PollPageProps) {
     },
   });
 
-  return { poll, statements };
+  const flaggedStatements = await prisma.flaggedStatement.findMany({
+    where: {
+      poll_id: poll.id,
+    },
+  });
+
+  const statementsWithFlaggedStatements = statements.map((statement) => ({
+    ...statement,
+    flaggedStatements: flaggedStatements.filter(
+      (flaggedStatement) => flaggedStatement.statementId === statement.id,
+    ),
+  }));
+
+  const filteredStatements = statementsWithFlaggedStatements.filter(
+    (statement) => statement.flaggedStatements.length > 1,
+  );
```

Comment:
I think it would be more performant to use the include and where params of prisma instead of this.

Reply:
Also why do we need the statements that have at least 2 reports. I thought it was the other way around. Also I think there's some code on the frontend to handle this already.